app/main.py

```diff
@@ -1,6 +1,26 @@
+import os
+from datetime import datetime
+
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+URL = f"http://api.weatherapi.com/v1/current.json?key={API_KEY}&q={CITY}"
```

Comment:
```suggestion
URL = f"http://api.weatherapi.com/v1/current.json
```

use const without external constants here